pkg/acs/calacs/acscte/dopcte.c

```diff
@@ -12,7 +12,7 @@
 #include "pcte.h"
 
 
-static int get_amp_array_size(const ACSInfo *acs, SingleGroup *x,
```

Comment:
Yes, so that it can be called by both algorithms. The gen2 exists in a different file.